src/AnnotatedRoutes/src/Annotation/Route.php

```diff
@@ -28,10 +28,10 @@ final class Route
     public $route;
 
     /**
-     * @Attribute(name="name", type="string", required=true)
+     * @Attribute(name="name", type="string")
      * @var string
```

Comment:
`@var string|null`

Reply:
Fixed

Reply:
Ah, yes, there is a problem with union types and doctrine annotations:
```
 [Type Error] Attribute "name" of @Route declared on method Spiral\Tests\Router\App\Controller\HomeController::index() expects a(n) string|null, but got string. 
```
I guess I'll have to rollback this fix

Reply:
@vladgorenkin yep =(